e2e-tests/cypress/tests/integration/channels/messaging/message_draft_with_empty_post_spec.js

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Stage: @prod
+// Group: @channels @messaging
+
+import {verifyDraftIcon} from './helpers';
+
+import * as TIMEOUTS from '../../../fixtures/timeouts';
+
+describe('Message Draft With empty Post and File Attachments', () => {
+    let testTeam;
+    let testChannel;
+
+    before(() => {
+      // # Create new team and new user and visit off-topic
+        cy.apiInitSetup({loginAfter: true}).then(({team, channel}) => {
+            testTeam = team;
+            testChannel = channel;
+            cy.visit(`/${testTeam.name}/channels/off-topic`);
+        });
+    });
+
+    it('MM-49228_1 Not storing draft if it only contains whitespace', () => {
```

Comment:
@AbhinavVihan the E2E looks awesome. However this test key is not exactly the ticket number but the corresponds to the test key in https://github.com/mattermost/mattermost-test-management/tree/main/data/test-cases/channels/global-drafts
I would kindly request you to please add your manual test step over there and get the new test key and drop it in the above.

Reply:
What should be the name of the file (MM-T5435.md) or can it e random in here [https://github.com/mattermost/mattermost-test-management/tree/main/data/test-cases/channels/global-drafts](https://github.com/mattermost/mattermost-test-management/tree/main/data/test-cases/channels/global-drafts)

Reply:
In the test management repository it can be random like MM-XX when you write the tests someone from QA will assign the key in that reply after which you can update the test in there as well as here

Reply:
@AbhinavVihan let me know if you have any more question

Reply:
To move forward with this PR, let's do add of the test case in `test-management` separately.